libffi.nim

```diff
@@ -11,36 +11,15 @@
 
 when defined(windows):
   import os
-  # on Windows we don't use a DLL but instead embed libffi directly:
```

Comment:
Oh sorry, I wasn't aware we do that. Isn't that much better than relying on DLLs?

Reply:
Yeah. I tried so hard to get compiling the 64 bit version to work. Something is wrong with the assembly sources and I have no idea what.

Reply:
Still trying to get a system that works for building the windows code. 64 bit code just fails and it's not clear why.